normalizer_lib.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""Library of language-agnostic FST rewrite rules to normalize text."""
+
+import unicodedata
+from pynini import *
+from config import *
+
+LANGUAGE = af
+
+GRAPHEMES = LANGUAGE.GRAPHEMES
+
+INITIAL_PUNCTUATION = LANGUAGE.INITIAL_PUNCTUATION
+FINAL_PUNCTUATION = LANGUAGE.FINAL_PUNCTUATION
+OTHER_PUNCTUATION = union(r"\[", r"\]")
+PUNCTUATION = union(INITIAL_PUNCTUATION, FINAL_PUNCTUATION, OTHER_PUNCTUATION)
+NUMBERS = union("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
```

Comment:
For this purpose, this is fine, since we're not dealing with other numerical systems in these particular languages that we're working with, but as a note, this should be customizable in the language specific configurations as well, so that we can include locale specific digit sets :).

Reply:
This can easily go into the language-specific configs and called here as `NUMBERS = LANGUAGE.NUMBERS` like the graphemes, initial, and final punctuation.

Reply:
SGTM thanks!